llama_stack/distribution/stack.py

```diff
@@ -232,6 +232,16 @@ def get_stack_run_config_from_template(template: str) -> StackRunConfig:
     return StackRunConfig(**replace_env_vars(run_config))
 
 
+def check_api(api_str: str) -> Api:
```

Comment:
I don't think making a function is necessary. You could just do this inline:

```
if api_str not in Api:
   raise ValueError(f"invalid API ({api_str})")
```

anybody running tests with llama stack already knows which APIs are and where they are to be found